protocol/synthetix/storage.dump.sol

```diff
@@ -495,24 +495,18 @@ library Pool {
 library RewardDistribution {
     struct Data {
         address distributor;
-        RewardDistributionEntry.Data entry;
+        uint128 __slotAvailableForFutureUse;
```

Comment:
uint128 is the same size?

Reply:
No, the removed var is a bytes32 pointer, so I am effectively doing a non backwards compatible change here.

The var I keep there is to make use of the half word that's free from the next uint128.